baremetal/metal3machine_manager_test.go

```diff
@@ -1554,34 +1561,54 @@ var _ = Describe("Metal3Machine manager", func() {
 				Expect(err).NotTo(HaveOccurred())
 				Expect(m3mTemplate.Spec.NodeReuse).To(BeTrue())
 			}
+
+			if tc.Host != nil {
+				savedbmh := bmh.BareMetalHost{}
+				err = fakeClient.Get(context.TODO(),
+					client.ObjectKey{
+						Name:      tc.Host.Name,
+						Namespace: tc.Host.Namespace,
+					},
+					&savedbmh,
+				)
+				Expect(err).NotTo(HaveOccurred())
+				if tc.capm3fasttrack != "" {
+					Expect(Capm3FastTrack).To(Equal(tc.capm3fasttrack))
```

Comment:
Trying to understand what are we testing here? We are setting `Capm3FastTrack = tc.capm3fasttrack` few lines back.

Reply:
yes we are setting it earlier, if you look to full condition it is testing the cases when we  set  and we do not set the Capm3fasttrack value

Reply:
My point is this will always be true` Expect(Capm3FastTrack).To(Equal(tc.capm3fasttrack)) ` since we are setting `Capm3FastTrack = tc.capm3fasttrack` few lines back.